magma/lib/magma/actions/model_update_actions.rb

```diff
@@ -128,7 +128,7 @@ def update_model_tables
     end
 
     def restart_server
-      return if Magma.instance.test?
+      return if Magma.instance.test? || Magma.instance.server_pid.nil?
```

Comment:
Is it safe to simply return here if the server_pid is nil? We are still responsible for ensuring a restart even if we were unable to determine the pid...

Reply:
The use case I was targeting is the magma local client.  Perhaps I could signal by some other means here?